services/QuillLMS/app/controllers/teachers/classroom_manager_controller.rb

```diff
@@ -269,4 +300,25 @@ def teacher_or_public_activity_packs
     end
   end
 
+  def set_alternative_schools
+    @alternative_schools = School.where(name: ['home school', 'us higher ed', 'international', 'not listed', 'other'])
+    @alternative_schools_name_map = {
+      'home school': 'Home school',
+      'us higher ed': 'U.S. higher education institution',
+      'other': 'Other',
+      'international': 'International institution',
+      'not listed': 'U.S. K-12 school'
+    }.stringify_keys
+  end
+
+  def get_account_info
+    account_info = current_user.generate_teacher_account_info
+    if account_info[:school] && account_info[:school].name
```

Comment:
Maybe move this school type logic into the `generate_teacher_account_info` method.